src/api/spec/bootstrap/features/webui/maintenance_workflow_spec.rb

```diff
@@ -32,7 +32,11 @@
     sleep 1 # Needed to avoid a flickering test.
     expect(page).to have_text('Source')
 
-    click_button('Accept')
+    within('#branch-modal .modal-footer') do
+      click_button('Accept')
+    end
+    sleep 1 
```

Comment:
what is that sleep good for? capybara will reload the page until it has the text.

Reply:
without sleep fails. with sleep passes.

Reply:
this is a super dangerous myth that will lead to tons of sleeps everywhere. Rule of thumb: if you don't understand why a sleep helps, it's the wrong fix :)